vignettes/Data_retrieval.Rmd

```diff
@@ -649,3 +651,143 @@ tcplPlot(lvl = 5, # data level
+- Restrictive clearance indicates that chemical bound to protein is relatively unavailable for hepatic metabolism or renal excretion (whereas non-restrictive clearance assumes that chemical bound to protein rapidly disassociates from that protein for metabolism and excretion). 
+
+```{r httk-aed, warning=FALSE, message=FALSE, eval=FALSE}
+# Generate AEDs
+toxcast_aed_liver_summary <- toxcast_mc5_liver_summary %>% 
+        summarize(aed.p5ac50.hu.css.50 = calc_mc_oral_equiv(conc=p5.ac50uM, 
+                                             dtxsid = 'DTXSID7021106', 
+                                             which.quantile=c(0.95),
+                                             species='Human',
+                                             restrictive.clearance=T, 
+                                             output.units='mgpkgpday',
+                                             model='3compartmentss'),
+        aed.p50ac50.hu.css.50 = calc_mc_oral_equiv(conc=p50.ac50uM, 
+                                              dtxsid = 'DTXSID7021106', 
+                                              which.quantile=c(0.95),
+                                              species='Human',
+                                              restrictive.clearance=T, 
+                                              output.units='mgpkgpday',
+                                              model='3compartmentss'),
+        aed.meanac50.hu.css.50 = calc_mc_oral_equiv(conc=mean.ac50uM, 
+                                               dtxsid = 'DTXSID7021106', 
+                                               which.quantile=c(0.95),
+                                               species='Human',
+                                               restrictive.clearance=T, 
+                                               output.units='mgpkgpday',
+                                               model='3compartmentss'))
+  
+```
+
+### Compare POD-Traditional with POD-NAM
+
+POD-Traditional (LEL and LOAEL) and POD-NAM (ToxCast-derived AEDs for 5%, 50%, and mean AC50 values) can be compared having been converted to mg/kg/day units. 
+``` {r compare, echo=FALSE}
+#create comparison table
+POD <- c("ToxRefDB LEL", 
+            "ToxRefDB LOAEL",
+            "ToxCast AED at 5th percentile AC50", 
+            "ToxCast AED at 50th percentile/median AC50", 
+            "ToxCast AED at mean AC50")
+Value <- c("1.5",
+          "1.5", 
+          "2.273744",
+          "7.666872", 
+          "16.09772")
+
+Table <- data.frame(POD, Value)
+
+datatable(Table,
+          filter='top',
+          options=list(pageLength = 15,searching=FALSE, autoWidth=FALSE,  scrollX=TRUE, initComplete = JS(
+            "function(settings, json) {",
+            "$('body').css({'font-family': 'Calibri'});",
+            "}"
+          )))
+```
+
+For this "Pentachlorophenol liver toxicity" example, POD-Traditional is more protective than the most sensitive POD-NAM.
```

Comment:
Suggested rewording: "For the "Pentachlorophenol liver toxicity" example provided here, the POD estimated from ToxRefDB (POD-Traditional) is more protective compared to the lowest summary estimate from ToxCast (POD-NAM)."